kubernetes/kube-state-metrics-deployment.yaml

```diff
@@ -28,6 +28,6 @@ spec:
             memory: 30Mi
             cpu: 100m
           limits:
-            memory: 50Mi
+            memory: 2Gi
```

Comment:
Sorry. I'll remove it. For some reason are happen a memory leak here. But I'll test more and open another issue about it.